src/code/z_scene_table.c

```diff
@@ -2348,9 +2348,8 @@ void func_8009FE58(GlobalContext* globalCtx) {
         temp = 0.020000001f;
         func_800AA76C(&globalCtx->view, Math_Coss(D_8012A39C) * (0.00009587531f * temp),
                       Math_Sins(D_8012A39C) * (0.00009587531f * temp), Math_Sins(D_8012A3A0) * (0.00009587531f * temp));
-        func_800AA78C(&globalCtx->view, 1.0f + (Math_Sins(D_8012A3A0) * (0.79999995f * temp)),
-                      1.0f + (Math_Coss(D_8012A3A0) * (0.39999998f * temp)),
-                      1.0f + (Math_Coss(D_8012A39C) * (1 * temp)));
+        func_800AA78C(&globalCtx->view, 1 + (Math_Sins(D_8012A3A0) * (0.79999995f * temp)),
+                      1 + (Math_Coss(D_8012A3A0) * (0.39999998f * temp)), 1 + (Math_Coss(D_8012A39C) * (1 * temp)));
```

Comment:
This is a non matching and these changes don't seem to have anything to do with skelanime. Are they intended?
And in any case it would probably be better to keep them separate.

Reply:
Ah, I was working on matching that function and must have accidentally done it on this branch. I'll revert.